modules/apps/batch-engine/batch-engine-test/src/testIntegration/resources/com/liferay/batch/engine/internal/test/dependencies/batch10/batch10/workflow-content.json

```diff
@@ -0,0 +1,437 @@
+{
+	"#child-nodes": [
```

Comment:
This file is also very confusing. Why don't you just create a file with the whole Workflow definition?